### PR TITLE
Feat/add styles

### PR DESCRIPTION
**Pull request by julizet**
Adds a combination of materials, styles and colors, that fit to each other.

In total we have now 9 materials to choose from.

### CODE REVIEW

File: lib/collection.ts [hunk 1]
```diff
@@ -9,3 +9,53 @@ export class Collection {
...
+	"ornamental curvilinear lines": ["black and white"],
+	"flat woodblock prints": ["black and white"],
+	"digital art": ["vibrant colors", "lgbtiq colours"],
+	"art deco": ["vibrant colors", "lgbtiq colours"], 
+	"cubism": ["raw light pastel colours", "contrast-rich and monochromatic"],
+	"folk art": ["symbolic colours", "saturated pigments", "eye-catching colour palette"]
+
+}
+
+//pleas write a function that first selects a random object key from materials, then get rendom style from material key and then gets a random color from style key
```
Comment (Jaszkowic) [inline lib/collection.ts:45]: Minor: I would remove that comment.

**Comment by Jaszkowic** [comment]
Looks good to me, only one minor comment.

**Review by Jaszkowic — APPROVED** [review]

File: lib/collection.ts [hunk 2]
```diff
@@ -9,3 +9,52 @@ export class Collection {
 		return this.list[Math.floor(Math.random() * this.list.length)];
 	}
 }
+
+const materials = {
+	"A dripping liquid oil painting": ["realistic", "modern"],
```
Comment (ff6347) [inline lib/collection.ts:14]: This creates an error

```ts
import { materials, styles } from "../lib/collection";

const material__dripping_liquid_oil_painting =
	materials["dripping liquid oil painting"];
console.log(material__dripping_liquid_oil_painting);

const style_from_material__modern = material__dripping_liquid_oil_painting[1];
console.log(style_from_material__modern);

const color_from_style__modern = styles[style_from_material__modern];
console.log(color_from_style__modern);

```

**Review by ff6347 — CHANGES_REQUESTED** [review]
There are mappings missing. I just found one. There might be more.

File: lib/collection.ts (hunk 2 above)
Comment (Jaszkowic) [reply lib/collection.ts:14]: @julizet  I think that's because the additionally added "a" or "an" in the material keys

Comment (julizet) [reply lib/collection.ts]: it was indeed missing. I also found that "conceptual art" was missing and added both within the latest commit

**Pull request merged**